release-notes.txt

```diff
@@ -1,3 +1,46 @@
+OpenMDAO Version 1.1.0 Alpha Release Notes
+September 22, 2015
+
+API Changes
+* Renamed 'ParamComp' to 'IndepVarComp,'' because it’s really just a component that
+supplies one or more independent variables as outputs, and it has no parameters. And
+overuse of term 'param' may have been causing confusion.
+
```

Comment:
What about the name change of the Driver methods that were mentioned in the proposed blog post?   I would at least mention that Driver.add_param changed to Driver.add_desvar, since that impacts users directly.  The other name changes really only matter to driver developers.